modules/govuk_jenkins/manifests/jobs/publishing_api_republish_organisations.pp

```diff
@@ -0,0 +1,22 @@
+# == Class: govuk_jenkins::jobs::publishing_api_republish_organisations
+#
+# Create a jenkins job to periodically run rake for the following tasks:
+# - represent_downstream:high_priority_document_type[:document_type]
+#
+# === Parameters:
+#
+
+class govuk_jenkins::jobs::publishing_api_republish_organisations(
+  $app_domain = hiera('app_domain'),
+) {
+
+  $check_name = 'publishing-api-republish-organisations'
```

Comment:
I don't think we need any of these variables as they are related to Icinga checks which we don't have for this job.

Reply:
Thanks @thomasleese ! I have updated this now.